content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -266,6 +266,7 @@ different Kubernetes components.
 | `EvenPodsSpread` | `true` | Beta | 1.18 | 1.18 |
 | `EvenPodsSpread` | `true` | GA | 1.19 | - |
 | `ExecProbeTimeout` | `true` | GA | 1.20 | - |
+| `ExternalPolicyForExternalIP` | `true` | GA | 1.18 | - |
```

Comment:
Hi @wangyysde .
Checking if the version is correct?

April 2020: https://github.com/kubernetes/kubernetes/pull/90537/files
March 2020: https://github.com/kubernetes/kubernetes/pull/88786/files


```
ExternalPolicyForExternalIP:                    {Default: true, PreRelease: featuregate.GA}, // remove in 1.20
```